src/SDKs/_metadata/powerbidedicated_resource-manager.txt

```diff
@@ -1,11 +1,11 @@
-2017-11-08 22:41:50 UTC
+2018-08-01 08:20:52 UTC
 
 1) azure-rest-api-specs repository information
 GitHub user: Azure
-Branch:      current
-Commit:      5e44f1a49008963f1b189dbd91903a22c470f757
+Branch:      master
+Commit:      1e2f0dae5c44d55b642d27232f9dcd7c0876488a
 
 2) AutoRest information
 Requested version: latest
-Bootstrapper version:    C:\Users\sagis\AppData\Roaming\npm `-- autorest@2.0.4166  
-Latest installed version:    2.0.4168
+Bootstrapper version:    C:\Program Files\nodejs `-- autorest@2.0.4283  
```

Comment:
Please pull down latest psSdkJson6 changes and run `msbuild build.proj` and regenerate the code

Reply:
Still waiting on this